api/serviceregistry/serviceregistry.proto

```diff
@@ -0,0 +1,48 @@
+syntax = "proto3";
+
+
+package svreg;
+option go_package = "serviceregistry";
+
+
+service Registry{
+  rpc RegisterWorkload (ServiceWorkload) returns (Empty) {} // Request register of a new service
+  rpc QueryService (ServiceRequest) returns (ServiceWorkloads) {} // Request a services
+  rpc RemoveWorkload (ServiceWorkload) returns (Empty) {} // Remove service from list
+}
+
+message Empty {}
+
+//The request used to register a new service or update the existing ones
+message ServiceWorkload {
```

Comment:
Do we really need both a singular and a plural for this? I think the plural can be sufficient. Either that, or return a list of singulars. there is no point in having two API objects with 90% the same code.

Reply:
done